### DISCUSSION TITLE
Error during visualization

### DISCUSSION THREAD

**Discussion by RogueW911** [tudelft/AE4353-Y24#4] [Ex_1]
When I'm trying to visualize the trajectory, there is a javascript error and the output of notebook cell reads:
WARNING:root:Timed out waiting for viewer to become ready. Make sure: https://app.rerun.io/version/0.18.2/widget.js is accessible.
If not, consider setting `RERUN_NOTEBOOK_ASSET`. Consult https://pypi.org/project/rerun-notebook/0.18.2/ for details.
I can open the first link in Chrome and I've tried a few ways suggested by Copliot but it didn't solve. I'm working on WSL2. Thank you.

**Comment by Huizerd** [comment]
could you try setting `RERUN_NOTEBOOK_ASSET` to the various options suggested at https://pypi.org/project/rerun-notebook/0.18.2/? you can set this variable like this:
![image](https://github.com/user-attachments/assets/ebd6da22-c716-4ebc-96bf-17f807636cd5)
and then I'd try the options `inline` and `serve-local`.

**Reply by RogueW911** [reply]
Today I tried another time and it worked without any change to the original code, but if it happens again next time, I'll try your method. Thank you!

**Reply by MarcColl10** [reply]
For me, it does not work with any method. It keeps giving a Javascript error.
` %env RERUN_NOTEBOOK_ASSET=serve-local
import rerun as rr
import rerun.blueprint as rrb `

**Comment by Huizerd** [comment]
Rerun issue opened: https://github.com/rerun-io/rerun/issues/7388
